src/utils/cron.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable import/extensions */
+/* eslint-disable import/no-useless-path-segments */
+/* eslint-disable node/no-missing-require */
+/* eslint-disable import/no-unresolved */
+
+const mongoose = require('mongoose');
+const cron = require('node-cron');
+const sendEmail = require('../utils/mailer');
+const logger = require('../utils/logger');
+const Project = require('../models/project');
+
+const today = new Date();
+
+async function initializeCronTasks() {
+  cron.schedule(
+    '0 9 * * *',
+    async () => {
+      const date = new Date();
+      date.setDate(date.getDate() - 1);
+      const projectTitles = await Project.find({
+        createdAt: { $gte: date },
+      }).select({ titles: 1 });
+      const htmlTemplate = `<p>${projectTitles.join('\n')}</p>`;
```

Comment:
I know I just requested these changes but this is still wrong, which is my bad. So, please accept my apologies.

`projectTitles` will return an array looks like `[{title: 'project title'}...]`. You can't use `join` in this file structure. You also need to select email addresses of users and add them into an array so you can pass this array to sendMail function.